icu4j/main/classes/core/src/com/ibm/icu/impl/number/LongNameHandler.java

```diff
@@ -26,7 +26,7 @@
 
     private static final int DNAM_INDEX = StandardPlural.COUNT;
     private static final int PER_INDEX = StandardPlural.COUNT + 1;
-    private static final int ARRAY_LENGTH = StandardPlural.COUNT + 2;
+    protected static final int ARRAY_LENGTH = StandardPlural.COUNT + 2;
```

Comment:
This line is "protected" rather than "private" because MixedUnitLongNameHandler is referencing it.